spec/features/admins/merchant_downgrade_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+###  Add comnfirmation of content appearing on page
+RSpec.describe 'role downgrade' do
+  context 'as an admin' do
+    it 'admin-users can downgrade the role of merchants' do
```

Comment:
Test looks good @mikekarnes123 ! One thing I was thinking could be added based on a user experience, 
- after the admin downgrades the merchant to user, 
- you could visit the items_index path
- and expect the page to not have content of those merchants item names

Reply:
Done!